.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
+
+on:
+  pull_request:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
I think the whole block should be intended two spaces more.
But no big deal now.